src/services/assetService.ts

```diff
@@ -102,7 +102,15 @@ export class AssetService {
      * Get assets from provider
      */
     public async getAssets(): Promise<IAsset[]> {
-        return await this.assetProvider.getAssets();
+        const assets = await this.assetProvider.getAssets();
```

Comment:
I would like to see a new unit test to validate this behavior before you check-in